test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/CpsPackageReferenceProjectTests.cs

```diff
@@ -3900,7 +3900,6 @@ public async Task GetTransitivePackagesAsync_WithTransitivePackageReferences_Ret
                 await result.CommitAsync(logger, CancellationToken.None);
                 ProjectPackages packages = await project.GetInstalledAndTransitivePackagesAsync(includeTransitiveOrigins: false, CancellationToken.None);
                 DateTime lastWriteTime = File.GetLastWriteTimeUtc(lockFilePath);
-                File.WriteAllText(lockFilePath, "** replaced file content to test cache **");
```

Comment:
This was causing that the assets file couldn't be read because it is an invalid JSON and there is a new call to load the JSON in https://github.com/NuGet/NuGet.Client/pull/5117/files#diff-cf69ebf676e61580349108d38e5dab1bbff91468194c6a8d342149d264139b15R192